plugin/restore/mysql/mysql.go

```diff
@@ -567,49 +646,45 @@ func (r *Restore) downloadBinlogFile(ctx context.Context, binlog mysql.BinlogFil
 	cmd.Stderr = os.Stderr
 
 	log.Debug("Downloading binlog files using mysqlbinlog", zap.String("cmd", cmd.String()))
-	resultFilePath := filepath.Join(resultFileDir, binlog.Name)
+	resultFilePath = filepath.Join(resultFileDir, binlogFileToDownload.Name)
```

Comment:
Is it correct to remove the resultFilePath?

Reply:
Reverted.
Second thought is that explicitly removing files at each error branch is better.